src/game.cpp

```diff
@@ -71,7 +71,14 @@
 #include <tchar.h>
 #endif
 
-#define dbg(x) DebugLog((DebugLevel)(x),D_GAME) << __FILE__ << ":" << __LINE__ << ": "
+inline std::ostream& dbg(int x)
+{
+  return DebugLog((DebugLevel)(x), D_GAME) << __FILE__ << ":" << __LINE__ << ": ";
```

Comment:
No. This will not work. Now the `__FILE__` and `__LINE__` will *always* refer to this line 76. Keeping that in the macro evaluates correctly to the line of the macro invocation.